source/modules/assets/cartridges.rst

```diff
@@ -0,0 +1,59 @@
+Cartridges
+==========
+
+This functionality of `Asset` module allows to create cartridges models and corresponding cartridges.
+
+In a cartridge form, the following informations are available:
```

Comment:
```suggestion
In a cartridge form, the following information are available:
```

Reply:
Should be?
In a cartridge form, the following information is available:
ref: https://blog.harwardcommunications.com/2010/11/09/information/

Reply:
Honestly... I do not know :D Maybe @cconard96 could tell us?

Reply:
from @fdechelle: «Since uncountable nouns are singular, they also require singular verbs. If you’re ever trying to decide whether to write the information is or the information are, remember that information is an uncountable noun and therefore needs is.» - https://www.grammarly.com/blog/countable-and-uncountable-nouns/

Reply:
Here is what I had for this in a local WIP:
"In a cartridge's form, several pieces of information are available"
I think "several pieces of information" is valid with "are" since it focuses on "pieces" and not "information". Without it, it does look like it needs to be "information is"

Reply:
Switched to "information is"